addons/strings.yaml

```diff
@@ -112,3 +112,26 @@ tryFirefoxExtension:
   downloadFirefox:
     value: Download Firefox
     comment: Button label
+
+upgradeToBundle:
+  upgrade:
+    value: Upgrade
+    comment: Text for upgrade button
+  title:
+    value: 3 tools, 1 privacy plan – VPN, email masking, and data protection
+    comment: Title for message
+  line1:
+    value: For $8.25/month, get our privacy tools in one plan.
+    comment: First line of message
+  bulletIntro:
+    value: "Plan includes:"
+    comment: Header for bullet list
+  bullet1:
+    value: VPN for secure, private browsing
+    comment: bullet focusing on VPN
+  bullet2:
+    value: Relay’s unlimited email masks to prevent spam spam and trackers
```

Comment:
```suggestion
    value: Relay’s unlimited email masks to prevent spam and trackers
```

(sorry, but couldn't miss)

![spam](https://github.com/user-attachments/assets/975da952-5402-4551-a0d3-baf4f7fd6363)

Reply:
Thank you, good catch. That was a copy/paste error from the mocks.